README.md

```diff
@@ -1 +1,29 @@
 # sftp-to-s3
+
+#### example
+
+```javascript
+const SftpToS3 = require('sftp-to-s3');
+const config = {
+  host: process.env.hostname,
+  username: process.env.user, 
+  password: process.env.password, 
+  aws: {
+    bucket: "bucket_name", accessKeyId: 'AKID', secretAccessKey: 'SECRET', region: 'us-west-2'
+  },
+  fileDownloadDir: "./",
+  completedDir: './completed-uploads/'
+};
```

Comment:
they arnt temp files, they are streams.  The only temp files we have are converted into arrays.